CriticalMapsUITests/SnapshotHelper.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  SnapshotHelper.swift
+//  Example
+//
+//  Created by Felix Krause on 10/8/15.
+//
+
+// -----------------------------------------------------
+// IMPORTANT: When modifying this file, make sure to
+//            increment the version number at the very
+//            bottom of the file to notify users about
+//            the new SnapshotHelper.swift
+// -----------------------------------------------------
```

Comment:
Is there a Swift package or any other keep the connection to automatically get updates in the future?

Reply:
Mhm... I don't think so. I didn't find anything about it online. But when one UITest runs in the CI, the CI would fail if someone has updated fastlane but not the SnapshotHelper. 

Because I forgot to update the SnapshotHelper every time in one of my projects, I added a script `update-dependencies.sh` or whatever name you like, where I collected all commands you have to run to update the dependencies. So you can run `bundle update` and `bundle exec fastlane snapshot update` in there. But yeah, this isn't really nice as well..

Reply:
Sounds like a good tradeoff. 

Created an issue for that #142